packages/web/utils/menuLinks.ts

```diff
@@ -160,6 +160,13 @@ export const MenuSectionLinks: MenuLinkSet[] = [
           'https://meta-game.notion.site/e0e83b7ea2a54d6294c5e167ba7b306a?v=81e5d4755ab44c3a93df3eaaee6fd369',
         icon: 'quests',
       },
+      // {
```

Comment:
Seems like we should just replace the other Quests link. For me it is login-gated anyway

Reply:
Would it make sense to also seed the quests in that other page?

Reply:
hmmm the [link](https://meta-game.notion.site/e0e83b7ea2a54d6294c5e167ba7b306a?v=81e5d4755ab44c3a93df3eaaee6fd369) of the current menu item doesn't lead me anywhere, actually, so we might as well just go for it, yeap.